### PR TITLE
Alcohol Tweaks

### PR DESCRIPTION
**Pull request by NanakoAC**
changes: 
  - bugfix: "Fixed dizziness effects on alcohol, psilocybin, and cryptobiolin taking a long to start up and sometimes never starting for low doses."
  - tweak: "Sip size from alcohol bottles is now the same as for glasses, which is half what it was."
  - tweak: "Rebalanced all alcoholic drinks with more believable alcohol values, and adjusted alcohol metabolism. Generally drinks are stronger but metabolise more slowly, pace yourself!"
  - tweak: "Drinking now causes temporary clumsiness until you sober up. Please don't drink and operate heavy machinery."
  - tweak: "Excessive drinking now has a chance to cause vomiting."
  - rscadd: "Different species now have varying susceptibility to alcohol. Tajarans get drunk slightly faster, skrell are twice as fast as humans, unathi can drink more, and vaurca get drunk very slowly, but alcohol poisons them."
  - bugfix: "Dousing people in alcohol and setting them on fire, now only works with spirits and liqeurs stronger than 40% ABV, and the heat of the resulting fire is based on the strength."
  - tweak: "Ethylredoxrazine now removes alcohol from the patient's blood and stomach, and decreases their intoxication. A large enough dose will make them completely sober."
  - tweak: "Ethylredoxrazine now metabolises and does its effects much more slowly."
  - tweak: "Coffee now sobers up drunk people a little."

### CODE REVIEW

File: code/modules/mob/living/carbon/human/intoxication.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#define		AE_DIZZY 		5
+#define 	AE_SLURRING 	10
+#define 	AE_CONFUSION 	18
+#define 	AE_BLURRING 	30
+#define 	AE_DROWSY 		55
```
Comment (Bedshaped) [inline code/modules/mob/living/carbon/human/intoxication.dm]: Can you add a define for like AE_VOMIT which gives you a chance of vomiting at around 55-80?

Comment (NanakoAC) [inline code/modules/mob/living/carbon/human/intoxication.dm]: Wow, i'm an idiot, how did i not think of that

**Comment by NanakoAC** [comment]
I have a little more to add to this, development flag for now

**Comment by NanakoAC** [comment]
Updated with a couple more features, more or less done now

File: code/modules/mob/living/carbon/human/intoxication.dm [hunk 2]
```diff
@@ -0,0 +1,66 @@
+#define		AE_DIZZY 		5
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/intoxication.dm]: What are these tabs.

**Comment by skull132** [comment]
My only thing is. Move clumsy to be a more severe side-effect.

Some effects from that mutation can near-instakill you. And if it's literally the second effect, theeen. Welp, RIP. Basically, there should be more warning before that.

**Comment by Bedshaped** [comment]
I would remove clumsy tbh until the effects and damage with objects are
better refined.
On 10 Jul 2016 21:51, "skull132" <notifications@github.com> wrote:

My only thing is. Move clumsy to be a more severe side-effect.

Some effects from that mutation can near-instakill you. And if it's
literally the second effect, theeen. Welp, RIP. Basically, there should be
more warning before that.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/Aurorastation/Aurora.3/pull/530#issuecomment-231610172>,
or mute the thread
<https://github.com/notifications/unsubscribe/ACPxPEPgFqitA2JMAILmJx5cukhP7u0Hks5qUVtRgaJpZM4JFufn>
.

**Comment by skull132** [comment]
Something needs to be done about clumsy. Either move it to be one of the later ones, or remove it for the time being.

**Comment by NanakoAC** [comment]
some changes made

**Pull request merged**